crates/sui-core/src/event_handler.rs

```diff
@@ -57,7 +57,7 @@ impl EventHandler {
         let envelopes = res?;
 
         // Ingest all envelopes together at once (for efficiency) into Event Store
-        self.event_store.add_events(&envelopes).await?;
+        self.event_store.add_tx_events(&envelopes).await?;
```

Comment:
I prefer to keep the old name `add_events` ... the reason is because I was going to refactor the event ingestion so add_events batches events from multiple TXes... this will make the ingestion much faster.  

Obviously with your change this will be much harder, until we fix the batch streaming system.